nixos/modules/services/web-servers/caddy/vhost-options.nix

```diff
@@ -41,8 +41,11 @@ in
         <emphasis>Note that this option does not create any certificates, nor
         does it add subdomains to existing ones – you will need to create them
         manually using <xref linkend="opt-security.acme.certs"/>. Additionally,
-        you should probably add the <literal>caddy</literal> user to the
-        <literal>acme</literal> group to grant access to the certificates.</emphasis>
+        you should probably set the group of corresponding acme cert to
+        <literal>caddy</literal>. If multiple services use the same cert, you can
+        add the <literal>caddy</literal> user to the <literal>acme</literal> group
```

Comment:
Yeah exactly this. We can help them along by defaulting to the `caddy` group for the required certs, as described in my other comment.